tensorflow/python/framework/test_util.py

```diff
@@ -27,6 +27,7 @@
 
 import numpy as np
 import six
+import tensorflow as tf
```

Comment:
I don't think this will work -- this will create a circular dependency since tf needs test_util.  You'll have to get the internal test_util module using the fully qualified path.